### PR TITLE
Removed dependency of lodash + minor

### PR DESCRIPTION
**Pull request by simplenotezy**
As discussed in https://github.com/EvanAgee/vuejs-wordpress-theme-starter/issues/99, removed dependency on lodash and saved around 526kb of space.

Also:

- Converted double qoutes to single qoutes
- Removed some unused variables

image

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import _ from "lodash";
-import axios from "axios";
-import SETTINGS from "../settings";
+// import _ from 'lodash';
```
Comment (EvanAgee) [inline src/api/index.js]: let's go ahead and remove anything that's commented out.

Comment (simplenotezy) [reply src/api/index.js]: Fixed

File: src/router/index.js [hunk 2]
```diff
@@ -39,24 +38,24 @@ const router = new Router({
...
+  },
 });
 
-router.afterEach((to, from) => {
+router.afterEach((to) => { // (to, from)
   // Add a body class specific to the route we're viewing
-  let body = document.querySelector("body");
-  let bodyClasses = body.className.split(" ");
+  let body = document.querySelector('body');
+  // let bodyClasses = body.className.split(' ');
```
Comment (EvanAgee) [inline src/router/index.js]: Let's remove this commented out code as well

File: src/router/index.js [hunk 3]
```diff
@@ -39,24 +38,24 @@ const router = new Router({
...
 
-  if (bodyClasses.length > 0) {
-    const newBodyClasses = bodyClasses.filter(theClass =>
-      theClass.startsWith("vue--page--")
-    );
-  }
+  // if (bodyClasses.length > 0) {
```
Comment (EvanAgee) [inline src/router/index.js]: Let's remove this commented out code as well

Comment (simplenotezy) [reply src/router/index.js]: Fixed

**Pull request merged**